.github/workflows/backend-ci-cd.yml

```
@@ -0,0 +1,103 @@
+name: backend-CI/CD
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)